docs/usage-and-performance-tools/usage-analytics.md

```diff
@@ -1,376 +1,376 @@
-
-Everyone in your Metabase, including deactivated accounts. Fields include:
-
-- User ID
-- Email
-- First Name
-- Last Name
-- Full Name
-- Date Joined
-- Last Login
-- Updated At
-- Is Admin
-- Is Active
-- SSO Source
-- Locale
-
-### Dashboard subscriptions model
-
-Which subscriptions are active, who created them, who's subscribed to them, when they're sent, and more.
-
-- Entity ID
-- Entity Qualified ID
-- Created At
-- Updated At
-- Creator ID
-- Archived
-- Dashboard Qualified ID
-- Schedule Type
-- Schedule Day
-- Schedule Hour
-- Recipient Type
-- Recipients
-- Recipient External
-- Parameters
-
-### Dashboard cards model
-
-Each row is a dashboard card: either a question card or a text card. Fields include:
-
-- ID
-- Dashboard ID
-- Dashboardtab ID
-- Question ID
-- Created At
-- Updated At
-- Size X
-- Size Y
-- Visualization Settings
-- Parameter Mappings
-
-## Databases model
-
-Information about your connected data sources. Fields include:
-
-- Entity ID
-- Entity Qualified ID
-- Created At
-- Updated At
-- Name
-- Description
-- Database Type
-- Metadata Sync Schedule
-- Cache Field Values Schedule
-- Timezone
-- Is On Demand
-- Auto Run Queries
-- Cache Ttl
-- Creator ID
-- Db Version
-
-## Tables model
-
-List of all tables across all connected data sources. Fields include:
-
-- Entity ID
-- Entity Qualified ID
-- Created At
-- Updated At
-- Name
-- Display Name
-- Description
-- Active
-- Database ID
-- Schema
-- Is Upload
-
-## Fields model
-
-All fields from all connected data sources. Fields include:
-
-- Entity ID
-- Entity Qualified ID
-- Created At
-- Updated At
-- Name
-- Display Name
-- Description
-- Base Type
-- Visibility Type
-- Fk Target Field ID
-- Has Field Values
-- Active
-- Table ID
+---
+title: Usage analytics
+---
+
+## Usage analytics
+
+{% include plans-blockquote.html feature="Metabase analytics" %}
+
+The **Metabase Analytics** collection is a special collection that contains view-only questions, dashboards, and models that help you understand how people are using your Metabase. These resources are useful for:
+
+- **Understanding Usage**: Understand how people use your Metabase (e.g., new questions, most active people and groups, and so on).
+- **Auditing activity**: Know who viewed or did what and when, including tracking dashboard and question views, queries, downloads, and other activity like changing settings or inviting people to your Metabase.
+- **Improving operations**: Know the slowest dashboards and questions, how your database's are performing, who's consuming the most resources, and so on.
+
+Some things to keep in mind with this special Metabase analytics collection:
+
+- Metabase analytics is a view-only collection. Even admins can't curate it. It is eternal.
+- By default, only admins and people in groups with [monitoring access](../usage-and-performance-tools/usage-analytics.md) can view the Metabase analytics collection (though admins can grant other groups view access to it).

```

Comment:
This is slightly imprecise, but not sure how to solve this...

People that had `monitoring` permission before 48 will gain access to the `Metabase analytics` collection when upgrading to 48.

After that, these two are completely unrelated.

So people in groups with monitoring access will only gain access if they had this permission before the upgrade and changing it after the upgrade won't grant/revoke access.